test/cypress/e2e/unit_tests_app_code.spec.cy.js

```diff
@@ -12,3 +14,61 @@ describe('Unit Test Application Code', function () {
     });
   });
 });
+
+describe('Component Boilerplate function', function () {
+  const componentName = 'TestComponent';
+  const componentFolder = 'global';
+  const importPath = `components/${componentFolder}/${componentName}`;
+
+  before(() => {
+    // Pre-test setup to ensure a clean state and run the script
+    cy.exec(
+      `npx tsc src/utils/create_component_file/index.ts && node src/utils/create_component_file/index.js ${componentName} ${componentFolder}`,
```

Comment:
Instead of `node ...` use `npx` please.

Reply:
This is not a simple replacement - `npx` fails here, because it's function is to execute binaries from `node_modules`. What is the reason for not using node command?